.changeset/thick-snakes-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/odata-service-inquirer': minor
```

Comment:
```suggestion
'@sap-ux/odata-service-inquirer': minor
'@sap-ux/inquirer-common': patch
```

Reply:
It will get bumped automatically, as I understand so not necessary?